### DISCUSSION TITLE
Possibility of a NetCDF engine

### DISCUSSION THREAD

**Discussion by richardyorke** [ornladios/ADIOS2#3821] [General]
Hi ORNL

I was wondering if there is any intention to build upon the work done to have a HDF5 engine and develop a NetCDF engine to write in that format? 

This would be handy for people in meteorology.

Thanks,
Richard.

**Comment by eisenhauer** [comment]
Hi Richard, and sorry for the delay in getting you a response.   The short story is that we're always interested in more use cases and getting into more user communities, but we also have to think strategically about what we do because of limited resources.  So, we'd be curious to hear more about where you think this sort of thing might have impact, details of the community it would benefit, etc.  Can you elaborate?
